README.md

```diff
@@ -83,7 +83,7 @@ KeyFactory algorithms:
 # Compatibility & Requirements
 ACCP has the following requirements:
 * JDK8 or newer (This includes both OracleJDK and [Amazon Corretto](https://aws.amazon.com/corretto/))
-* 64-bit Linux running on x86_64 (also known as x64 or AMD64)
+* 64-bit Linux or MacOs running on x86_64 (also known as x64 or AMD64)
```

Comment:
It works on Linux and MacOS running on aarch64 also, doesn't it? :-)

Reply:
presumably. we know that it works on aarch64 linux, but we haven't yet tested it yet on aarch64 MacOS as far as I know (I use an x86 mac, and it's unclear what the underlying architecture is for the MacOS jobs).

in any case, I wanted to keep the README guidance on architecture and OS hewed to whatever is available in maven until we publish an external release of ACCP 2.0. until then, maven-vended ACCP will still only be supported on x86 linux.